sources/packages/forms/Dockerfile

```diff
@@ -4,10 +4,13 @@
 
 # Use Node image, maintained by Docker:
 # hub.docker.com/r/_/node/
-FROM docker-remote.artifacts.developer.gov.bc.ca/node:12.2.0-alpine
+FROM docker.io/library/node:lts-alpine3.10
```

Comment:
Please define a specific tag for the node image as it was before. We must have control over these changes, right?